server/events/project_command_runner.go

```diff
@@ -125,29 +125,30 @@ type JobURLSetter interface {
 	SetJobURLWithStatus(ctx models.ProjectCommandContext, cmdName models.CommandName, status models.CommitStatus) error
 }
 
-//go:generate pegomock generate -m --use-experimental-model-gen --package mocks -o mocks/mock_job_message_sender.go JobMessageSender
+//go:generate pegomock generate -m --use-experimental-model-gen --package mocks -o mocks/mock_job_closer.go JobCloser
 
-type JobMessageSender interface {
-	Send(ctx models.ProjectCommandContext, msg string, operationComplete bool)
+// Job Closer closes a job by marking op complete and clearing up buffers if logs are successfully persisted
+type JobCloser interface {
+	CloseJob(jobID string)
```

Comment:
What does closing a job mean from a grammar standpoint 👀  I don't really have a better name off the top of my head, but just wanted to call it out to make sure we're always thinking about logical naming.

Reply:
Yeah I'm struggling to come up with a better name 😅 What do you think of 
`type JobFinisher interface{
          FinishJob(jobID string)
}
`

Reply:
Sounds like something from mortal kombat lol

Reply:
Haha I'll leave it as `JobCloser` for now then!